docs/Options-in-Constructor.md

```diff
@@ -485,14 +486,27 @@ You can play with a live Plunker sample:
 <a id="compress"></a>
 ## Document compression
 
-Use the [compress](https://github.com/surveyjs/survey-pdf/blob/1220a71b51daddf1c4c8d506382c50be5f1b2941/src/doc_controller.ts#L48) option to specify whether to compress the  generated PDF document. A compressed document does not support using [custom fonts](https://surveyjs.io/Documentation/Pdf-Export?id=Customization-ChangeFonts#use-custom-font).
+Use the [compress](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L48) option to specify whether to compress the  generated PDF document. A compressed document does not support using [custom fonts](https://surveyjs.io/Documentation/Pdf-Export?id=Customization-ChangeFonts#use-custom-font).
 
-- [compress](https://github.com/surveyjs/survey-pdf/blob/1220a71b51daddf1c4c8d506382c50be5f1b2941/src/doc_controller.ts#L48)  
+- [compress](https://github.com/surveyjs/survey-pdf/blob/master/src/doc_controller.ts#L48)  
 `compress?: boolean;`
 
-```JavaScript
+```js
 var options = {
     compress: true
 };
 var survey = new SurveyPDF.SurveyPDF(json, options);
 ```
+
+## Image Rendering
+
+[Image questions](https://surveyjs.io/Documentation/Library?id=questionimagemodel) have an [imageFit](https://surveyjs.io/Documentation/Library?id=questionimagemodel#imageFit) property that specifies how images should fit into their container. If exported images should take this property into account, enable the `applyImageFit` option. Please note that with this option, the quality of images may be lower because they pass through several conversions.
```

Comment:
```suggestion
[Image questions](https://surveyjs.io/Documentation/Library?id=questionimagemodel) have an [imageFit](https://surveyjs.io/Documentation/Library?id=questionimagemodel#imageFit) property that specifies how images should fit into their containers. If exported images should take this property into account [should take this property into account?], enable the `applyImageFit` option. Please note that with this option, the quality of images may be lower because they pass through several conversions.
```

Reply:
```suggestion
[Image questions](https://surveyjs.io/Documentation/Library?id=questionimagemodel) have an [imageFit](https://surveyjs.io/Documentation/Library?id=questionimagemodel#imageFit) property that specifies how images should fit into their containers. If exported images should apply this property, enable the `applyImageFit` option. Please note that with this option, the quality of images may be lower because they pass through several conversions.
```

Reply:
Roman, this work. :)